anyway/infographics_utils.py

```diff
@@ -1841,6 +1818,8 @@ def get_request_params(
     if all(value is None for value in location_info.values()):
         return None
 
+    # TODO: this does not return the latest accident date for tables which are not AccidentMarkerView. For example, in the
+    # InjuredAccidentsWithPedestriansWidget, where we query against InvolvedMarkerView
```

Comment:
These dates are now aligned and we [added a data test](https://github.com/hasadna/anyway-etl/pull/6) in our etl to verity the non of the tables has missing data.

Reply:
removed TODO